### PR TITLE
Feature/el03 riley

### PR DESCRIPTION
**Pull request by kbeling4**
Included interface for Riley cross section data.

### CODE REVIEW

File: src/ACEtk/interpretation/EL03/RileyCrossSection.hpp
Comment (david8dixon) [inline src/ACEtk/interpretation/EL03/RileyCrossSection.hpp]: Fails to link when using ```RileyCrossSection::params_per_energy``` rather than ```num_params```

**Review by david8dixon — CHANGES_REQUESTED** [review]
Would be nice to resolve constexpr weirdness before merging. @apmccartney 

Why is the testing broken? @jlconlin

File: src/ACEtk/interpretation/EL03/RileyCrossSection.hpp
Comment (apmccartney) [inline src/ACEtk/interpretation/EL03/RileyCrossSection.hpp]: What lead you to choose to repeat this in the two methods rather than define once at the class level?

File: src/ACEtk/interpretation/EL03/test/RileyCrossSections.test.cpp
Comment (apmccartney) [inline src/ACEtk/interpretation/EL03/test/RileyCrossSections.test.cpp]: prefer `.front` to explicit indexing

**Review by apmccartney — APPROVED** [review]
Contingent on @david8dixon verifying format usage and the travis build passing. Good Work Kyle!

**Comment by apmccartney** [comment]
You have an uncommitted source file named `ACEtk/src/ACEtk/interpretation/EL03/RileyCrossSections.hpp` (notice the plural) in your source tree. Please remove it and regenerate the Cmake files using metaconfigure

**Review by apmccartney — CHANGES_REQUESTED** [review]
Will approve when tests pass

**Review by apmccartney — APPROVED** [review]

**Comment by apmccartney** [comment]
@david8dixon This is now waiting on your approval

**Comment by apmccartney** [comment]
@kbeling4 You can see your test coverage results here: https://coveralls.io/builds/18045318/source?filename=src/ACEtk/interpretation/EL03/RileyCrossSection.hpp

**Review by david8dixon — APPROVED** [review]

**Pull request merged**
